### DISCUSSION TITLE
DSEP BPP Documentation Repository

### DISCUSSION THREAD

**Discussion by shuklaharsha** [beckn/DSEP-Specification#106] [General]
Hi @ravi-prakash-v!
Please take a look at the link to the documents below.
Jobs and Internships: https://docs.google.com/document/d/1DZXVv-dahHheKF4BxVg6gvROPaZDfjHMbTn7GTK4_zw/edit#heading=h.o6c62vw2l1mi
Tutoring and Mentorship: https://docs.google.com/document/d/1XjRuvDs_4_gGQv7ap07WNDFbC00Ulx1y8zNfp8hYh_4/edit
Scholarships and Grants: https://docs.google.com/document/d/1iOfW5Q0o38W52QfYe6GdnVQ_loMltrLCpt2eNN2rxic/edit
Training and Courses: https://docs.google.com/document/d/10WxKoIGdWYlT4ZGtIldyRy-NrKlE0WkDI8WOal-AEfw/edit

**Comment by nirmay** [comment]
Thanks for putting this together, reference apps are really important for those developers/organisations starting with DSEP.

I looked through the docs and here are some of my thoughts
- The docs looked detailed, in spite they follow the template, they provide different depth of information, some work is needed on the template itself.
- The docs seem to assume comprehensive understanding of beckn protocol, may be when they are offered as a part of starter kit, they are better aligned with introduction to FIDE and Beckn protocol in general.
- They jump directly into implementation, since each of these docs are for separate domains, It would be useful to set better context at the beginning for each of these domains. e.g. job and internship could have a separate section (or a separate document) explaining the core Entities, i.e. Jobs and how that is populated in the lifecycle of creation, applications, offer etc, mapping each of them through various beckn apis. Also, it could include an entity or state flow diagram to explain this in detail.
- It is important to understand what is mandated by the protocol and what is a scoping decision by the reference (or any) app. Without this clarity, we run the risk of having apps that look like the same, which does not make sense.

Once I have access to comment on docs, I can add more minor comments on the doc itself, but It would be great for us to get started on above points from the overall flow perspective.
